mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+            ncFile.createDimension('nxAtlantic', len(mocLatRegional))
+            ncFile.createDimension('nz', len(refTopDepth))
+            # create variables
+            latBinGlobal = ncFile.createVariable(
+                'latBinGlobal', 'f4', ('nxGlobal',))
+            latBinGlobal.description = \
+                'latitude bins for MOC Global streamfunction'
+            latBinGlobal.units = 'degrees (-90 to 90)'
+            latBinAtlantic = ncFile.createVariable(
+                'latBinAtlantic', 'f4', ('nxAtlantic',))
+            latBinAtlantic.description = \
+                'latitude bins for MOC Atlantic streamfunction'
+            latBinAtlantic.units = 'degrees (-90 to 90)'
+            depth = ncFile.createVariable('depth', 'f4', ('nz',))
+            depth.description = 'depth'
+            depth.units = 'meters'
+            fldGlobal = ncFile.createVariable(
+                'mocGlobal', 'f4', ('nz', 'nxGlobal'))
+            fldGlobal.description = \
+                'MOC Global streamfunction, annual climatology'
+            fldGlobal.units = 'Sv (10^6 m^3/s)'
+            fldAtlantic = ncFile.createVariable(
+                'mocAtlantic', 'f4', ('nz', 'nxAtlantic'))
+            fldAtlantic.description = \
+                'MOC Atlantic streamfunction, annual climatology'
+            fldAtlantic.units = 'Sv (10^6 m^3/s)'
+            # save variables
+            latBinGlobal[:] = mocLatGlobal
+            latBinAtlantic[:] = mocLatRegional
+            depth[:] = refTopDepth
+            fldGlobal[:, :] = mocGlobal
+            fldAtlantic[:, :] = mocRegional
+            ncFile.close()
+        else:
+            # Read from file
+            print '   Read previously computed MOC streamfunction from file...'
+            ncFile = netCDF4.Dataset(outputFileClimo, mode='r')
+            mocLatGlobal = ncFile.variables['latBinGlobal'][:]
+            mocLatRegional = ncFile.variables['latBinAtlantic'][:]
+            refTopDepth = ncFile.variables['depth'][:]
+            mocGlobal = ncFile.variables['mocGlobal'][:, :]
+            mocRegional = ncFile.variables['mocAtlantic'][:, :]
+            ncFile.close()
+
+        print '   Plot climatological global MOC...'
+        title = 'Global MOC (ANN, years {:04d}-{:04d})\n {}'.format(
+                 startYearClimo, endYearClimo, mainRunName)
+        regionName = 'global'
+        figureName = '{}/moc_{}_{}_years{:04d}-{:04d}.png'.format(
+                      plotsDirectory, regionName, mainRunName,
+                      startYearClimo, endYearClimo)
+        contourLevels = config.getExpression(
+                            sectionNameClimo,
+                            '{}ContourLevels'.format(regionName),
+                            usenumpyfunc=True)
+        colorbarLevels = config.getExpression(
+                            sectionNameClimo,
+                            '{}ColorbarLevels'.format(regionName))
+
```

Comment:
I think this is where the colormapName code, etc, as highlighted above, should go.